app/infrastructure/database/repositories.py

```
@@ -0,0 +1,265 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional, TypeVar, Generic
+from uuid import UUID
+from datetime import datetime, timezone
+from models import *
```

Ruff (0.11.9)

`from models import *` used; unable to detect undefined names

(F403)